recipes/pcre2/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "pcre2" %}
+{% set version = "10.30" %}
+{% set sha256 = "b549873a39f804480c2e6145a78adcba53e38162d90ef6ea92384f6ecf2fde76" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  command:
+    - pcre2test -version
+
+about:
+  home: http://www.pcre.org/
+  license: PCRE2
+  license_family: BSD
+  license_file: LICENCE
+  summary: 'The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5.'
```

Comment:
Would shorten this a bit. This longer form could be added as `description` with line wrapping.